sdk/cosmosdb/cosmos/test/public/functional/item.spec.ts

```diff
@@ -236,12 +236,17 @@ describe("bulk/batch item operations", function () {
         },
         throughput: 25100,
       });
+      afterEach(async function () {
+        //console.log(diagnosticToString);
+      });
+
       readItemId = addEntropy("item1");
-      await container.items.create({
+      const itemResponse = await container.items.create({
         id: readItemId,
         key: "A",
         class: "2010",
       });
+      console.log(itemResponse.cosmosDiagnostics);
```

Comment:
asserts on the diagnostics to validate correctness? Also, please add test-coverage for calling itemResponse.comsosDiagnostics more than once to ensure it doesn't vanish after the first call.